source/imaer-shared/src/test/resources/json/NcaCustomCalculationPoint.json

```diff
@@ -13,9 +13,17 @@
     "assessmentCategory" : "HUMAN_HEALTH",
     "roadLocalFractionNO2" : 0.423,
     "entityReferences" : [ {
+      "description" : "Description of habitat",
       "code" : "Habitat code",
+      "entityType" : "HABITAT_TYPE",
+      "criticalLevels" : {
+        "NOX_CONCENTRATION" : 1.23,
+        "NH3_CONCENTRATION" : 4.56,
+        "EC_CONCENTRATION" : 7.89
+      }
+    }, {
       "description" : "Description of habitat",
-      "entityType" : "HABITAT_TYPE"
+      "entityType" : "CRITICAL_LEVEL_ENTITY"
     } ]
```

Comment:
guess it works, but conceptually it might make sense to have the critical levels in the second object.